src/main/java/seedu/address/logic/parser/AddEventCommandParser.java

```diff
@@ -0,0 +1,53 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_DATETIME;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_DESCRIPTION;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_FRIEND_NAME;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
+
+import java.util.Set;
+
+import seedu.address.logic.commands.AddEventCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.event.DateTime;
+import seedu.address.model.event.Event;
+import seedu.address.model.person.Description;
+import seedu.address.model.person.Name;
+
+/**
+ * Parses input arguments and creates a new AddEventCommand object
+ */
+public class AddEventCommandParser implements Parser<AddEventCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the AddEventCommand
+     * and returns an AddEventCommand object for execution.
+     * @throws ParseException if the user input does not conform the expected format
+     */
+    public AddEventCommand parse(String args) throws ParseException {
```

Comment:
Should include handling of more invalid values and throw a `ParseException` whenever one is found (e.g. empty lines, etc.) - we can find out more with more testing in the next push.